### PR TITLE
Update Ubuntu Build and Install Instructions

### PR DESCRIPTION
**Pull request by jonnius**

### CODE REVIEW

**Comment by jonnius** [comment]
I just figured you might want to remove Ubuntu 16.04 instructions completely as they will not meet the dependency requirements:
* CMake 3.5.2 (3.13 required)
* Qt 5.9 (5.10 required)

**Comment by deepbluev7** [comment]
Yeah, we may want to do that. Do you want to also make that change in this PR or should I merge it as is?

**Comment by jonnius** [comment]
Just changed the PR to bring the README up-to-date in regards of Ubuntu.

File: README.md [hunk 1]
```diff
@@ -179,22 +179,14 @@ sudo pacman -S qt5-base \
...
-sudo apt-get update
-sudo apt-get install -y g++-7 qt59base qt59svg qt59tools qt59multimedia cmake liblmdb-dev libsodium-dev
-```
-
-##### Ubuntu 19.10
+##### Ubuntu 20.04
 
 ```bash
 # Build requirements + qml modules needed at runtime (you may not need all of them, but the following seem to work according to reports):
-sudo apt install g++-7 cmake liblmdb-dev libsodium-dev libssl-dev qt{base,declarative,tools,multimedia,script,quickcontrols2-}5-dev qml-module-qt{gstreamer,multimedia,quick-extras,-labs-settings,graphicaleffects,quick-controls2} libqt5svg5-dev
```
Comment (deepbluev7) [inline README.md:196]: Did you intentionally remove the qml modules? Those are needed at runtime. So while you can build it without them, you will get a blank window! :D

Comment (jonnius) [reply README.md:196]: You are right, the chat window remains empty without them. I wonder why nheko from Ubuntu repos works just fine [without any qml module installed](https://packages.ubuntu.com/focal/nheko).

**Comment by deepbluev7** [comment]
Thanks, looks good. Thank you for fixing up our stale documentation :3

**Pull request merged**